src/main/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthResponse.java

```diff
@@ -57,6 +58,8 @@
     int initializingShards = 0;
     int unassignedShards = 0;
     int numberOfPendingTasks = 0;
+    int numberOfFetchShardStarted = 0;
```

Comment:
yea, its the in flight ones, I will rename it to in flight and keep it simple and not separate to 2 types